GVFS/GVFS.Common/PlaceholderListDatabase.cs

```diff
@@ -260,7 +260,7 @@ public void WriteAllEntriesAndFlush(IEnumerable<IPlaceholderData> updatedPlaceho
 
         private IEnumerable<string> GenerateDataLines(IEnumerable<IPlaceholderData> updatedPlaceholders)

```

Comment:
FYI - On master the placeholders are now being stored in a SQLite database (see #1140).  When updating the Linux feature branch to pick up that change you'll need to make the table creation call aware of whether the platform is case sensitive (currently the table is created with a case-insensitive path/key).